utils/userStatus.js

```diff
@@ -302,6 +302,52 @@ const getNextDayTimeStamp = (timeStamp) => {
   return nextDateDateTime.getTime();
 };
 
+/**
+ * Converts a timestamp to either the start or end of a day in UTC time.
+ *
+ * @param {number} timestamp - The timestamp to be converted.
+ * @param {boolean} isEndOfDay - A flag indicating whether to convert to the end of the day (true) or the start of the day (false).
+ * @returns {number} The converted timestamp.
+ */
+const convertTimestampToUTCStartOrEndOfDay = (timestamp, isEndOfDay) => {
```

Comment:
the function does not check if the from and until timestamps are valid before converting them to UTC. This means that the function could throw an error if the timestamps are invalid.

Reply:
Thanks for bringing this to notice. Will add safety check

Reply:
Changes applied. Please review again @iamitprakash